psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -291,38 +417,245 @@ private static void createLine(PsseNonTransformerBranch psseLine, ContainersMapp
+            double squareMod = sumR * sumR + sumX * sumX;
+
+            double r1 = ((r31 * r12 - x31 * x12) * sumR + (r31 * x12 + r12 * x31) * sumX) / squareMod;
+            double x1 = ((r31 * x12 + r12 * x31) * sumR - (r31 * r12 - x31 * x12) * sumX) / squareMod;
+
+            double r2 = ((r12 * r23 - x12 * x23) * sumR + (r12 * x23 + r23 * x12) * sumX) / squareMod;
+            double x2 = ((r12 * x23 + r23 * x12) * sumR - (r12 * r23 - x12 * x23) * sumX) / squareMod;
+
+            double r3 = ((r23 * r31 - x23 * x31) * sumR + (r23 * x31 + r31 * x23) * sumX) / squareMod;
+            double x3 = ((r23 * x31 + r31 * x23) * sumR - (r23 * r31 - x23 * x31) * sumX) / squareMod;
+
+            //set a voltage base at star node with the associated Zbase
+            double v0 = 1.0;
+            double zbV0 = Math.pow(v0, 2) / perUnitContext.getSb();
+
+            ThreeWindingsTransformerAdder tfoAdder = voltageLevel1.getSubstation().newThreeWindingsTransformer()
+                    .setRatedU0(v0)
+                    .setId(id);
+
+            ThreeWindingsTransformerAdder.LegAdder l1adder = tfoAdder.newLeg1();
+            l1adder.setR(r1 * zbV0)
+                    .setX(x1 * zbV0)
+                    .setG(gmPu * w1 * w1 / zbV0)
+                    .setB(bmPu * w1 * w1 / zbV0)
+                    .setRatedU(voltageLevel1.getNominalV() * w1)
+                    .setConnectableBus(bus1Id)
+                    .setVoltageLevel(voltageLevel1Id);
+            l1adder.add();
+
+            ThreeWindingsTransformerAdder.LegAdder l2adder = tfoAdder.newLeg2();
+            l2adder.setR(r2 * zbV0)
+                    .setX(x2 * zbV0)
+                    .setG(0)
+                    .setB(0)
+                    .setRatedU(voltageLevel2.getNominalV() * w2)
+                    .setConnectableBus(bus2Id)
+                    .setVoltageLevel(voltageLevel2Id);
+            l2adder.add();
+
+            ThreeWindingsTransformerAdder.LegAdder l3adder = tfoAdder.newLeg3();
+            l3adder.setR(r3 * zbV0)
+                    .setX(x3 * zbV0)
+                    .setG(0)
+                    .setB(0)
+                    .setRatedU(voltageLevel3.getNominalV() * w3)
+                    .setConnectableBus(bus3Id)
+                    .setVoltageLevel(voltageLevel3Id);
+            l3adder.add();
+
+            ThreeWindingsTransformer tfo3W = tfoAdder.add();
+
+            if (psseTfo.getFirstRecord().getStat() == 1) {
+                tfo3W.getLeg1().getTerminal().connect();
+                tfo3W.getLeg2().getTerminal().connect();
+                tfo3W.getLeg3().getTerminal().connect();
+            }
+
+            //set the init value at the star point
+            tfo3W.setProperty(V_PROPERTY, Float.toString((float) psseTfo.getSecondRecord().getVmstar())); //TODO: check the right base to put the voltage module
+            tfo3W.setProperty(ANGLE_PROPERTY, Float.toString((float) psseTfo.getSecondRecord().getAnstar()));
```

Comment:
Is it OK to set these properties?

Reply:
We need it if we want to run a load flow from the starting values as the result values are given in the psse file.
if not, the load flow cannot run because the starting values at the star node will be undefined